pr2eus/pr2-interface.l

```diff
@@ -324,6 +325,82 @@ Example: (send self :gripper :rarm :position) => 0.00"
 	 (when (and current origin)
 	   (mapcar #'v- current origin))
        current)))
+  (:switch-controller
+   (stop start &key (strictness 1))
+   (let (req res start-controller stop-controller)
+     (ros::wait-for-service "/pr2_controller_manager/switch_controller")
+     (setq req (instance pr2_mechanism_msgs::SwitchControllerRequest :init))
+     (setq start-controller (cond (start start) (t "dummy_controller")))
```

Comment:
Is this correct usage to set `dummy_controller`? If you refer some other codes that use this usage, please write about this as docstring.